a11ym

```diff
@@ -169,54 +175,88 @@ var isErrorOrWarning = function (result) {
     return 'error' === result.type || 'warning' === result.type;
 };
 
-var hasErrors = false;
+var parseURL = function (url) {
+    var parsed = URL.parse(url);
+
+    if (!parsed.protocol) {
+        parsed.protocol = 'http';
+    } else {
+        parsed.protocol = parsed.protocol.replace(':', '');
```

Comment:
maybe document the why of this, it's not obvious if you never printed out the content of the var